FinancialTracker/Firebase.swift

```diff
@@ -66,20 +72,49 @@ class FirebaseHandler {
     }
 
     func signIn(email: String, password: String, completionHandler: @escaping (FirebaseError?, Bool) -> Void) {
-        Auth.auth().signIn(withEmail: email, password: password) { _, error in
+        auth.signIn(withEmail: email, password: password) { _, error in
 
-            if error != nil {
+            guard error == nil else {
                 completionHandler(FirebaseError.auth(error), false)
+                return
             }
             completionHandler(nil, true)
         }
     }
     
-    func signOut() {
+    func signOut(completionHandler: @escaping (FirebaseError?, Bool) -> Void) {
         do {
-            try Auth.auth().signOut()
+            try auth.signOut()
         } catch let signOutError {
```

Comment:
There is no point in catching the error if you are passing it on to the `completionHandler`. 
Are there other types of errors to expect?